### ISSUE TITLE
Javascript Frontend Tests

### ISSUE THREAD

**Issue by cain** [Automattic/liveblog#449]
Thoughts on creating some react tests? Possibly Jest? 😸

**Comment by philipjohn** [comment]
Yes! Happy to hear suggestions, as a more comprehensive test framework is something we need to dedicate some time to.
